src/main/java/org/sadun/util/polling/DirectoryPoller.java

```diff
@@ -533,7 +535,9 @@ private void runCycle() {
 
 				File fls[] = dir.listFiles(filter);
 				if (fls == null) {
-					System.err.println((new StringBuilder()).append("Warning: directory ").append(dir).append(" does not exist").toString());
+					String warningMsg = String.format("Warning: directory %s does not exist", dir.toString());

```

Comment:
No need for String.format(xxxxx) if this is using LoggerFactory, just logger.warn("whatever {}", variable)

Reply:
Fixed that and normalized some formatting.